src/sparse/KokkosSparse_sptrsv.hpp

```diff
@@ -58,6 +58,16 @@
 #include "KokkosSparse_sptrsv_symbolic_spec.hpp"
 #include "KokkosSparse_sptrsv_solve_spec.hpp"
 
+#ifdef KOKKOSKERNELS_ENABLE_TPL_SUPERLU
+ #include "slu_ddefs.h"
```

Comment:
It's a REALLY BAD IDEA to expose the SuperLU headers to downstream code.  SuperLU is known to be bad about defining really short macros like `MIN` that collide with users' code.  Please hide SuperLU includes in a `.cpp` file; don't put them in headers that applications can see!

In general, please avoid exposing TPL headers (especially for C libraries that tend to pollute the global namespace) in headers that applications might see.